src/components/ProjectWork/ContactSection.js

```diff
@@ -1,46 +1,88 @@
-import React from 'react';
-import { Button } from 'react-bootstrap';
-
-const Template = ({ orgName, projectTitle }) => `
-  Hello ${orgName},\n\n
-  My name is [...] and I am a [...]. I am reaching out to you because I found your ${projectTitle} project listing on the Build for Black Lives project website. My teammates (CC'd) and I would love to offer our [design/programming etc] skills to help work on your project pro bono.\n\n
-  <Please remember to include information such as who your team is, what your background and experience is, and any questions you may have about the project, logistical or otherwise>\n\n
-  Please feel free to look at examples of my work at [portfolio link]. You can contact me at [your email] or [your phone number]. Thank you very much and I hope to work with you soon!\n\n
-  Best,\n
-  [Your name]
-  `;
+import React, { useState } from 'react';
+import { Button, Card } from 'react-bootstrap';
+
+const Template = ({ orgName, projectTitle }, textType) => {
```

Comment:
is there a reason textType is outside of the rest of the props desctructuring?

Reply:
textType isn't a part of the project object we're passing in but another param to distinguish spacing for the website and mailto template